examples/compositor.c

```diff
@@ -431,6 +437,31 @@ static void handle_cursor_button(struct wl_listener *listener, void *data) {
 		event->button, event->state);
 }
 
+static void handle_tool_axis(struct wl_listener *listener, void *data) {
+	struct sample_state *sample =
+		wl_container_of(listener, sample, tool_axis);
+	struct wlr_event_tablet_tool_axis *event = data;
+	if ((event->updated_axes & WLR_TABLET_TOOL_AXIS_X) &&
```

Comment:
Can we get a case where either one or the other axis is updated?

Reply:
Not that I've ever observed. I suppose it could happen in theory.